tests/integration/targets/hcloud_subnetwork/defaults/main/main.yml

```diff
@@ -2,4 +2,20 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 ---
 hcloud_network_name: "{{ hcloud_ns }}"
-hetzner_vswitch_id: 43065
+hetzner_vswitch_ids:
+  - 43065
+  - 44166
+  - 44167
+  - 44168
+  - 44170
+  - 44171
+  - 44172
+  - 44173
+  - 44174
+  - 44175
+  - 44176
+  - 44177
+  - 44178
+  - 44179
+  - 44180
+  - 44181
```

Comment:
Is this done to avoid conflicts? If yes, could you please add a comment to indicate that.

Reply:
Yes, will add a comment, but still working on a way to prevent the CI from failing.